LICENSE.md

```diff
@@ -131,5 +137,5 @@ The Author must make the Mod source freely and publicly available.
 --------
 
 The Author may grant a User the status of Contributor or Author.  
-The Author retains the right to the Mod name.  
+The Author retains the right to the Mod's name, Assets, and all other .  
```

Comment:
"and all other"...?

Maybe go ahead and finish this statement?

As a side note, it'd be better to list additional rights on additional lines.